Index: out/production/Mini - MASSIVE/results.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Ticks;  Losing Team; Number of Drones Left Alive; Winning Drone Num; Losing Drone Num\r\n219, axis, 16, Leeroy Jenkins, Flank\r\n175, axis, 8, Leeroy Jenkins, Fire Teams\r\n166, axis, 9, Leeroy Jenkins, Leeroy Jenkins\r\n50, allies, 0, Leeroy Jenkins, Leeroy Jenkins\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/out/production/Mini - MASSIVE/results.txt b/out/production/Mini - MASSIVE/results.txt
--- a/out/production/Mini - MASSIVE/results.txt	(revision 9b805235a0e7352b0eff5bef351271078f66804c)
+++ b/out/production/Mini - MASSIVE/results.txt	(date 1619104964997)
@@ -3,3 +3,5 @@
 175, axis, 8, Leeroy Jenkins, Fire Teams
 166, axis, 9, Leeroy Jenkins, Leeroy Jenkins
 50, allies, 0, Leeroy Jenkins, Leeroy Jenkins
+193, axis, 16, Fire Teams, Unknown Algo
+159, axis, 16, Fire Teams, Fire Teams
Index: src/results.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Ticks;  Losing Team; Number of Drones Left Alive; Winning Drone Num; Losing Drone Num\r\n219, axis, 16, Leeroy Jenkins, Flank\r\n175, axis, 8, Leeroy Jenkins, Fire Teams\r\n166, axis, 9, Leeroy Jenkins, Leeroy Jenkins\r\n50, allies, 0, Leeroy Jenkins, Leeroy Jenkins\r\n193, axis, 16, Fire Teams, Unknown Algo\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/results.txt b/src/results.txt
--- a/src/results.txt	(revision 9b805235a0e7352b0eff5bef351271078f66804c)
+++ b/src/results.txt	(date 1619105010554)
@@ -4,3 +4,5 @@
 166, axis, 9, Leeroy Jenkins, Leeroy Jenkins
 50, allies, 0, Leeroy Jenkins, Leeroy Jenkins
 193, axis, 16, Fire Teams, Unknown Algo
+159, axis, 16, Fire Teams, Fire Teams
+150, axis, 24, Fire Teams, Fire Teams
